circuits/plonk/src/scalars.rs

```diff
@@ -38,7 +37,6 @@ impl<F: FftField> ProofEvaluations<Vec<F>> {
 }
 
 #[derive(Clone, Debug)]
-#[cfg_attr(feature = "ocaml_types", derive(ocaml::IntoValue, ocaml::FromValue))]
```

Comment:
Can you un-delete this, so that the existing bindings will still compile while you finish converting the new ones?

Reply:
I don't think this will work as the existing bindings use a different Field trait (from our zexe fork)

Reply:
Ah, of course.